blockbase/functions.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+if ( ! function_exists( 'blockbase_support' ) ) :
+	function blockbase_support()  {
+
+		// Alignwide and alignfull classes in the block editor
+		add_theme_support( 'align-wide' );
+
+		// Adding support for core block visual styles.
+		add_theme_support( 'wp-block-styles' );
```

Comment:
Should block-based themes opt-into the opinionated styles by default? (If not, this should be available as a theme.json setting)